libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -1,26 +1,32 @@
-import { Component, OnInit } from '@angular/core';
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
 import { Store } from '@ngrx/store';
 import {
   addToReadingList,
   clearSearch,
   getAllBooks,
+  getBooksError,
+  getBooksLoaded,
   ReadingListBook,
   searchBooks
 } from '@tmo/books/data-access';
 import { FormBuilder } from '@angular/forms';
 import { Book } from '@tmo/shared/models';
+import { Observable } from 'rxjs';
 
 @Component({
   selector: 'tmo-book-search',
   templateUrl: './book-search.component.html',
-  styleUrls: ['./book-search.component.scss']
+  styleUrls: ['./book-search.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class BookSearchComponent implements OnInit {
-  books: ReadingListBook[];
-
+  books$: Observable<ReadingListBook[]>;
   searchForm = this.fb.group({
     term: ''
   });
+  getBooksLoaded$: Observable<boolean> = this.store.select(getBooksLoaded);
+  getBooksError$: Observable<string> = this.store.select(getBooksError);
+  noResultFoundText = 'No Result Found !!';
```

Comment:
Make it as constant:
`ERROR_TEXT = 'No record found.';`

Reply:
Done